index.js

```diff
@@ -377,9 +377,7 @@ export default function contextMenu(options = {}) {
 			disposeMenu();
 		};
 
-		if (win.once !== undefined) { // Support for BrowserView
-			win.webContents.once('destroyed', disposable);
-		}
+		webContents(win).once('destroyed', disposable);
```

Comment:
`win` is not always `BrowserWindow`, so we must use utils to get `WebContents`: https://github.com/sindresorhus/electron-context-menu/blob/691c307dcedd18f4c02dd385e98610c3a8d20dd1/index.js#L7